test/components/tools/openapi/test_openapi_client_live.py

```diff
@@ -30,6 +30,7 @@ def test_serperdev(self, test_files_path):
         assert "invention" in str(response)
 
     @pytest.mark.integration
+    @pytest.mark.skip("This test hits rate limit on Github API. Skip for now.")
```

Comment:
Do these tests pass locally?

Reply:
Yes, but Github is a bit restrictive once it gets bombarded by all different OS/Python variants.